kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
```

Comment:
Right, I'm still wondering exactly how we'll bootstrap up the Preset table; I guess we can just manually construct it and then save a fixture. The main use for these constants will be in the special logic around determining which preset to use for ambiguous cases.

But, for clarity, it would be useful to have the constant names start with the kind name, so we would want:
`VIDEO_HIGH_RES`
`VIDEO_LOW_RES`

We'll also need to have separate thumbnail presets for any kind we want to support thumbnails for. So, we would have:

`DOCUMENT_THUMBNAIL`
`VIDEO_THUMBNAIL`
(and maybe etc)